src/NuGet.Clients/PackageManagement.UI/Xamls/InfiniteScrollList.xaml.cs

```diff
@@ -58,11 +58,7 @@ public bool CheckBoxesEnabled
             }
             set
             {
-                _checkBoxesEnabled = value;
-                _updateButtonContainer.Visibility =
-                    _checkBoxesEnabled ?
-                    Visibility.Visible :
-                    Visibility.Collapsed;
+                _checkBoxesEnabled = value;                
```

Comment:
Is it possible for CheckBoxesEnabled to be set but UpdateCheckBoxStatus() not be called? You have moved this logic to UpdateCheckBoxStatus().